src/media/js/site/login.js

```diff
@@ -25,10 +25,9 @@ export function loginRequired(Component, LoginHandler, group,
   class AuthenticatedComponent extends React.Component {
     render() {
       const isLoggedIn = !!this.props.user.token;
-      const hasPermission = (process.env.NODE_ENV === 'test' ||
-                             _checkPermissions(this.props.user, group));
+      const hasPermission = _checkPermissions(this.props.user, group);
 
-      if (isLoggedIn && hasPermission) {
+      if (process.env.NODE_ENV === 'test' || isLoggedIn && hasPermission) {
```

Comment:
I know, it just feels like an indication of a leaky abstraction if we're forced to litter our code with `if (process.env.NODE_ENV === 'test')` statements.

I'm not familiar enough with the stack to have a suggestion for something better, but I'd bet that there's something we can mock to avoid this.